<ui version="4.0" >
 <class>DialogWindow</class>
 <widget class="QDialog" name="DialogWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>329</height>
   </rect>
  </property>
  <widget class="QWidget" name="verticalLayout" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>561</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" >
    <item>
     <widget class="QTextEdit" name="textEdit" >
      <property name="verticalScrollBarPolicy" >
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="readOnly" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="lineEdit" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>290</y>
     <width>381</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton" >
   <property name="geometry" >
    <rect>
     <x>400</x>
     <y>290</y>
     <width>75</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text" >
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
